samples/sensor/wsen_hids/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: WSEN-HIDS Sensor Sample
+tests:
+  sample.sensor.wsen-hids:
+    harness: sensor
+    tags: sensors
+    depends_on: i2c hids
```

Comment:
make this `depends on: i2c` only.  the `hids` part will get covered by the `filter`

Reply:
Done.